currency_rate_update_VN_VCB/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================================================================================
+Currency Rate Update Joint Stock Commercial Bank for Foreign Trade of Vietnam - Vietcombank service
+===================================================================================================
+
+Download exchange rates automatically from
+Joint Stock Commercial Bank for Foreign Trade of Vietnam - Vietcombank (Vietcombank) service
+
+Installation
+============
+
+To install this module, you need to:
+
+    * clone the branch 11.0 of the repository https://github.com/OCA/l10n-vietnam
+    * add the path to this repository in your configuration (addons-path)
+    * update the module list
+    * search for "Currency Rate Update - VCB" in your addons
+    * install the module
+
+The module depends on currency_rate_update module available on https://github.com/OCA/currency
+
+Usage
+=====
+
+Go to the `Currency Usage <https://github.com/OCA/currency/tree/11.0/currency_rate_update>`_ for more detail.
```

Comment:
This module does not change usage, but only adds an additional source of exchange rates when configuring in: 
Accounting > Configuration > Multi-currencies > Rate Auto-download

Usage is the same as the module `Currency Rate Update <...>`.